src/config/config.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+
+@Injectable()
+export class ConfigAppService {
+    constructor(private readonly configService: ConfigService) {}
+
+    get(key: string): string {
+        return this.configService.get<string>(key);
+    }
+
+    getNumber(key: string): number {
+        return this.configService.get<number>(key);
+    }
```

Comment:
it's possible use typescript Generic types

```suggestion
    get<Type>(key: string): Type {
        return this.configService.get<Type>(key);
    }
```

Reply:
Will try the generics so.